### PR TITLE
Issue card number click should not open modal (just open page)

### PR DESCRIPTION
**Pull request by abierbaum**

### CODE REVIEW

File: src/app/issue/issue_ctrl.js [hunk 1]
```diff
@@ -85,6 +85,10 @@ mod.controller('IssueCtrl', function($scope, $modal, $rootScope, trRepoModel, gh
          $modal.open(opts);
       },
 
+      stopEvent: function(event) {
```
Comment (abierbaum) [inline src/app/issue/issue_ctrl.js]: I don't love the location of this method, but figured I would put it here until we need a more global place to put this.

File: src/app/issue/issue.tpl.html [hunk 2]
```diff
@@ -7,7 +7,14 @@
   <div ng-if="issueCtrl.isPullRequest()"
        class="build_header"></div>
   <div class="header">
-    <a href="{{issueCtrl.issue.html_url}}" title="Open on GitHub" class="number ng-scope"><i ng-if="issueCtrl.isPullRequest()" class="icon-code-fork"></i>{{issueCtrl.issue.number}}</a>  <span class="title ng-binding">
+    <a ng-href="{{issueCtrl.issue.html_url}}"
+       ng-click="$event.stopPropagation()"
```
Comment (abierbaum) [inline src/app/issue/issue.tpl.html:11]: This is the key call.

**Comment by abierbaum** [comment]
+1

File: src/app/issue/issue.tpl.html [hunk 3]
```diff
@@ -7,7 +7,14 @@
...
+       target="_blank"
+       title="Open on GitHub"
+       class="number ng-scope">
```
Comment (my8bird) [inline src/app/issue/issue.tpl.html:14]: we can kill the ng-scope in the class here

**Comment by my8bird** [comment]
+1, with the option to clean up the copy and paste from inspector

**Pull request merged**